builtin/providers/rancher/provider.go

```diff
@@ -71,7 +73,7 @@ func init() {
 
 		"secret_key": "API secret used to authenticate with the rancher server",
 
-		"api_url": "The URL to the rancher API",
+		"api_url": "The URL to the rancher API, must include version uri (ie. v1 or v2-beta)",
```

Comment:
I'm not sure to see the point in breaking compatibility here if you're going to use the v2 API everywhere anyway.